docs/references/api/tables_views.rst

```diff
@@ -193,7 +193,7 @@ The :code:`fts` filter mentioned above has a number of options to support flexib
 
   curl "http://localhost:3000/tsearch?my_tsv=not.wfts(french).amusant"
 
-Using `websearch_to_tsquery` requires PostgreSQL of version at least 11.0 and will raise an error in earlier versions of the database.
+This also works with columns of text and JSON types, converting them to ``tsvector`` accordingly.
```

Comment:
A subsection would be better for this. This way we can also highlight the feature on the release notes.

Maybe "automatic tsvector casting"`?